src/inttest/java/com/faforever/api/deployment/ExeUploadControllerTest.java

```diff
@@ -0,0 +1,51 @@
+package com.faforever.api.deployment;
+
+import com.faforever.api.AbstractIntegrationTest;
+import org.junit.Before;
+import org.junit.Test;
+import org.springframework.mock.web.MockMultipartFile;
+import org.springframework.test.context.jdbc.Sql;
+
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.fileUpload;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@Sql(executionPhase = Sql.ExecutionPhase.BEFORE_TEST_METHOD, scripts = "classpath:sql/prepClanData.sql")
+@Sql(executionPhase = Sql.ExecutionPhase.AFTER_TEST_METHOD, scripts = "classpath:sql/cleanClanData.sql")
+public class ExeUploadControllerTest extends AbstractIntegrationTest {
+  private MockMultipartFile file;
+
+  @Before
+  public void setUp() {
+    super.setUp();
+    file = new MockMultipartFile("file", "ForgedAlliance.exe", "application/octet-stream", ";fjas;fjas".getBytes());
```

Comment:
Maybe use a constant here that is a proper string as well (e.g. `"someBytes".getBytes()` or `new byte{ 1, 2 ,3, 4 }`

Reply:
new byte[]{ 1, 2 ,3, 4 }